config/environments/production.rb

```diff
@@ -28,27 +28,26 @@
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-#MYCODE
-config.action_mailer.delivery_method = :sendmail
-# Defaults to:
-# config.action_mailer.sendmail_settings = {
-#   location: '/usr/sbin/sendmail',
-#   arguments: '-i'
-# }
-config.action_mailer.perform_deliveries = true
-config.action_mailer.raise_delivery_errors = true
-config.action_mailer.default_options = {from: 'crickethowzat7@gmail.com'}
-
-config.action_mailer.delivery_method = :smtp
-config.action_mailer.smtp_settings = {
-  address:              'smtp.gmail.com',
-  port:                 587,
-  user_name:            'crickethowzat7@gmail.com',
-  password:             'howzathowzat',
-  authentication:       'plain',
-  enable_starttls_auto: true }
-
-config.active_storage.service = :amazon
+  config.action_mailer.delivery_method = :sendmail
```

Comment:
delivery_method can be either `:sendmail` or `:smtp`. Please use one and remove other